crates/router/src/connector/powertranz/transformers.rs

```diff
@@ -1,210 +1,332 @@
-use masking::Secret;
+use api_models::payments::{Address, Card};
+use common_utils::pii::Email;
+use diesel_models::enums::RefundStatus;
+use masking::{ExposeInterface, Secret};
 use serde::{Deserialize, Serialize};
+use uuid::Uuid;
 
 use crate::{
-    connector::utils::PaymentsAuthorizeRequestData,
+    connector::utils::{self, CardData},
+    consts,
     core::errors,
-    types::{self, api, storage::enums},
+    types::{self, api, storage::enums, transformers::ForeignFrom},
 };
 
-//TODO: Fill the struct with respective fields
-#[derive(Default, Debug, Serialize, Eq, PartialEq)]
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "PascalCase")]
 pub struct PowertranzPaymentsRequest {
-    amount: i64,
-    card: PowertranzCard,
+    transaction_identifier: String,
+    total_amount: f64,
+    currency_code: String,
+    three_d_s_ecure: bool,
```

Comment:
Is it supposed to be "ThreeDSEcure" or "ThreeDSecure"? If "ThreeDSecure" then it must be "three_d_secure".